README.md

```diff
@@ -273,96 +354,44 @@ network_connections:
     state: up
 ```
 
-As said, the `name` identifies a unique profile. However, you can refer to the
-same profile multiple times. Therefore it is possible to create a profile and
-activate it separately.
+Setting a connection profile activated:
 
-### `ip`
+```yaml
+network_connections:
+  - name: eth0
+    state: up
+```
 
-The IP configuration supports the following options:
+Setting a connection profile deactivated:
 
 ```yaml
 network_connections:
   - name: eth0
-    type: ethernet
-    ip:
-      route_metric4: 100
-      dhcp4: no
-      #dhcp4_send_hostname: no
-      gateway4: 192.0.2.1
+    state: down
+```
 
-      dns:
-        - 192.0.2.2
-        - 198.51.100.5
-      dns_search:
-        - example.com
-        - subdomain.example.com
+Creating a present connection profile:
 
-      route_metric6: -1
-      auto6: no
-      gateway6: 2001:db8::1
+```yaml
+network_connections:
+  - name: eth0
+    #persistent_state: present  # default
+    type: ethernet
+    autoconnect: yes
+    mac: 00:00:5e:00:53:5d
+    ip:
+      dhcp4: yes
+```
 
-      address:
-        - 192.0.2.3/24
-        - 198.51.100.3/26
-        - 2001:db8::80/7
+Setting an absent connection profile:
```

Comment:
It should read: "Deleting a connection profile", no? "Setting an absent connection profile" makes no sense IMO.